app/util.js

```diff
@@ -30,7 +30,8 @@ const specialPages = [
     "history",
     "newtab",
     "notifications",
-    "version"
+    "version",
+    "bookmarks"
```

Comment:
I think it makes sense, although not enforced, to have these alphabetically sorted.